frontend/src/screens/wallet/receive/ReceiveInvoice.tsx

```diff
@@ -150,7 +124,7 @@ export default function ReceiveInvoice() {
                       </div>
                       <div>
                         <Button onClick={copy} variant="outline">
-                          <CopyIcon className="w-4 h-4 mr-2" />
+                          <CopyIcon />
```

Comment:
<img width="1089" height="462" alt="image" src="https://github.com/user-attachments/assets/1d322d87-50bd-4a22-b61d-d3525977dc62" />

Spacing between label and inputs seem to be gone now